Form-Controls/index.html

```diff
@@ -1,28 +1,105 @@
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+      <form>
+        <fieldset class="details1">
+          <legend>choose your favorite T-shirt</legend>
+          <div>
+            <label for="black"
+              ><img
+                class="color_img"
+                alt="Black T-shirt"
+                src="https://static.thcdn.com/productimg/1600/1600/13254031-1024901106882108.jpg"
+              /><br />
+              <input
+                type="radio"
+                id="balck"
+                name="colour"
+                required
+              />Black</label
+            >
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+            <label for="grey"
+              ><img
+                class="color_img"
+                alt="Grey T-shirt"
+                src="https://static.thcdn.com/productimg/1600/1600/13934053-2014987380424392.jpg"
+              /><br /><input
+                type="radio"
+                id="grey"
+                name="colour"
+                required
+              />Grey</label
+            >
 
-</html>
\ No newline at end of file
+            <label for="green"
+              ><img
+                class="color_img"
+                alt="Green T-shirt"
+                src="https://static.thcdn.com/productimg/1600/1600/13934070-2814993600442706.jpg"
+              /><br /><input type="radio" id="green" name="colour" required />
+              Green</label
+            >
+          </div>
+          <label for="select_design"
+            >Size
+            <select name="size" id="select_design">
+              <option>XS</option>
+              <option>S</option>
+              <option>M</option>
+              <option>L</option>
+              <option>XL</option>
+              <option>XXL</option>
+            </select></label
+          >
+        </fieldset>
+
+        <fieldset class="details2">
+          <legend>Details for delivery</legend>
+          <label for="name"
+            >Full Name <br /><input
+              type="text"
+              id="name"
+              placeholder="your register name"
+              pattern="[Bb]oshra [Mm]ahmoudi|[Mm]ehreen [Aa]ziz|[Ss]hayan [Mm]ahnam|[Hh]ary [Ll]i"
```

Comment:
Took me a while to get this working on the site. Then I figured you have put 'hary' instead of 'harry'. Just becareful on typos and spellings, especially for rules and patterns. A common but can be devastating mistake. But very nicely done getting the pattern setup!

Reply:
sorry about that, spelling is one of my weakness in coding. I should pay attention more